src/mocks.ts

```diff
@@ -512,12 +512,11 @@ const globals = {
 
       IOGas = IOGas + key.length + (val?.length || 0);
 
-      // if (val === null) {
-      //   stateCache.delete(key);
-      // } else {
-      //   stateCache.set(key, val);
-      // }
-      tmpState.set(key, val!);
+      if (val === null || val === "null") {
+        tmpState.delete(key);
+      } else {
+        tmpState.set(key, val);
+      }
```

Comment:
```suggestion
      tmpState.set(key, val!);
```
Copy production implementation. Type of `tmpState` might need to be updated.